src/vsphere_cpi/Rakefile

```diff
@@ -0,0 +1,69 @@
+require 'time'
+require 'shellwords'
+
+# Remove did_you_mean as it's very annoying in a Rakefile
+DidYouMean::Correctable.send(:remove_method, :to_s) if defined? DidYouMean
+
+# Colorize error messages when the trace output is a TTY
+module ColorizeExceptionMessageDetails
+  def display_exception_message_details(ex)
+    return super unless (options.trace_output || $stderr)&.isatty
+    if ex.instance_of?(RuntimeError)
+      trace "\e[31;01m#{ex.message}\e[0m"
+    else
+      trace "\e[31;01m#{ex.class.name}: \e[0m#{ex.message}"
+    end
+  end
+end
+Rake::Application.send(:prepend, ColorizeExceptionMessageDetails)
+
+desc 'Create swagger client from swagger spec'
+task :swagger, :name do |t, args|
+
+  sh '/bin/bash', '-ec', [
+    "rm -rf",
+    "./lib/nsxt/*"
+  ].join(' ')
+
+  sh '/bin/bash', '-ec', [
+    "swagger-codegen generate",
+    "-i ./data/nsx_api.json",
+    "-l ruby",
+    "-o ./lib/nsxt",
+    "-c ./data/config.json -t ./data/swagger-nsxt-template"
+  ].join(' ')
```

Comment:
The `sh 'bin/bash', '-ec'` pattern is a hack that we had to use in the other Rakefile to get shell interpolation to work (`<()`). Here you can just do:

```ruby
sh 'swagger-codegen', 'generate', '-i', 'data/nsx_api.json', etc.
```

Reply:
Fixed